src/puya/awst_build/validation/scratch_slots.py

```diff
@@ -32,7 +32,7 @@ def invalid_slot_usages(self) -> Iterator[tuple[int, SourceLocation]]:
 
     def visit_contract_fragment(self, statement: awst_nodes.ContractFragment) -> None:
         super().visit_contract_fragment(statement)
-        # TODO: gather reserved from bases
+        # TODO : gather reserved from bases
```

Comment:
```suggestion
```

We are now doing this [here](https://github.com/algorandfoundation/puya/blob/a5466a0d9644b15049a567581f1081aa47868644/src/puya/awst_build/module.py#L195)